docs/resources/aws_kms_key.md

```diff
@@ -0,0 +1,175 @@
+---
+title: About the aws_kms_key Resource
+---
+
+# aws_kms_key
+
+Use the `aws_kms_key` InSpec audit resource to test properties of a single AWS KMS Key.
+
+AWS Key Management Service (AWS KMS) is a managed service that makes it easy for you to create and control the encryption keys used to encrypt your data. AWS KMS lets you create master keys that can never be exported from the service and which can be used to encrypt and decrypt data based on policies you define.
+
+Each AWS KMS Key is uniquely identified by its key-id or key-arn.
```

Comment:
Use the syntax used in the resource: `key_id` instead of `key-id`, and `arn` instead of `key-arn`.